src/Gantt.js

```diff
@@ -60,8 +60,23 @@ export default function Gantt(element, tasks, config) {
 	}
 
 	function reset_variables(tasks) {
+		/*
+		  According to http://snapsvg.io/docs/, the Snap constructor supports
+			usage of a string css selector or a DOM SVG element, thus we support both.
+		*/
+		if(typeof element === "string") {
+		    self.element = document.querySelector(element);
+		}
+		else if (element instanceof SVGElement) {
+		    self.element = element;
+		}
+		else if (element instanceof HTMLElement) {
+		    self.element = element.querySelector('svg');
```

Comment:
`self.element = element`

Won't work if the element is an HTML Element. I am currently developing an Angular application and it did not work with that code change. The line that I used:

`self.element = element.querySelector('svg');`

Works correctly when the element is an HTML element because the Snap library expects an SVG input only.